chain/chunks/src/lib.rs

```diff
@@ -743,6 +749,12 @@ impl ShardsManager {
 
             self.encoded_chunks.remove_from_cache_if_outside_horizon(&chunk_hash);
             self.requested_partial_encoded_chunks.remove(&chunk_hash);
+            info!(
+                "%%% care shard has_all_parts && has_all_receipts prev_block_hash {} chunk_hash {:?} shard_id {}",
```

Comment:
@SkidanovAlex I thought i traced "care shard" case here and "don't care" case above. i think there isn't a path not traced (trace them separately gives slightly more information)